linters/kubebench/spec/master.go

```diff
@@ -4,9 +4,17 @@ import (
 	"github.com/samber/lo"
 	batchv1 "k8s.io/api/batch/v1"
 	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+const (
+	limitCPU   = "200m"
+	limitMem   = "128Mi"
+	requestCPU = "100m"
```

Comment:
This is to large request, 100m is quite a lot for such job which mostly performs IO

Reply:
how much do you think will do?